src/backend/tcop/utility.c

```diff
@@ -2714,6 +2719,12 @@ CreateCommandTag(Node *parsetree)
 			tag = CMDTAG_COPY;
 			break;
 
+			/*
+		case T_AlterModuleRenameStmt:
```

Comment:
Leftover debugging code?

Reply:
commented out some actual functionality :) Removed the /* */